### DISCUSSION TITLE
HA switch to enable/disable streamer mode possible?

### DISCUSSION THREAD

**Discussion by thomaslunze** [revk/xESP32-Faikout#243] [General]
Hi revk
would it be possible to create an entity in HA (like demand control) to also be able to swich the streamer mode on/off?
Or is there a way already other than the MQTT command?

**Comment by revk** [comment]
If we do streamer, which we do, there should be a command anyway. I'll have to have a look.

**Reply by revk** [reply]
Ah, this may be a clue ``      // TODO comfort/streamer/sensor/quiet``

I'll have to see what I can do.

**Reply by revk** [reply]
OK, I'll put on a beta with a command of `streamer` and `sensor` with value of `0` or `1`.

**Reply by revk** [reply]
Beta done, but I have not made an HA command for it as such.
